packages/core/src/components/DataTable/index.tsx

```diff
@@ -370,19 +420,25 @@ export class DataTable extends React.Component<DataTableProps & WithStylesProps,
   rowGetter = (expandedDataList: ExpandedRow[]) => ({ index }: { index: number }) =>
     expandedDataList[index];
 
+  cache = new CellMeasurerCache({
+    fixedHeight: false,
+    fixedWidth: true,
+    defaultHeight: 1,
```

Comment:
should this be closer to reasonable height values? like `16` or so?

Reply:
Yeah, there was some point in debugging where this helped, but now I can't tell the difference between setting it at all. Going to leave it at 16.